README.md

```diff
@@ -1,50 +1,80 @@
-# Deno Standard Modules
+# Deno Standard Library
```

Comment:
Are we giving up on "module" as the official naming for Deno modules? If so, we should probably update /manual.

Reply:
Ah, you might be misinterpreting. The title change is because this project is the Deno Standard Library. It's as simple as that. My thinking is that otherwise, a newcomer might see the current title and wonder whether to refer to this as "Deno Standard Modules" or "Deno Standard Library". This is a consistency improvement.

Reply:
Ah alright, seems reasonable.